Samples/BlazorExample/BlazorExample/Client/Shared/TextInput.razor

```diff
@@ -0,0 +1,12 @@
+﻿<div>
+  <input @bind-value="Property.Value" @bind-value:event="oninput"
+         disabled="@(!Property.CanWrite)" /><br />
```

Comment:
If PropertyInfo exposed an IsValid property, you could add the following here, as a nice accessibility enhancement:

aria-invalid="Property.IsValid"

aria-invalid tells screen-readers to inform the user that the input is not valid